cmd/object-api-utils.go

```diff
@@ -191,7 +191,7 @@ func checkObjectNameForLengthAndSlash(bucket, object string) error {
 	if runtime.GOOS == globalWindowsOSName {
 		// Explicitly disallowed characters on windows.
 		// Avoids most problematic names.
-		if strings.ContainsAny(object, `:*?"|<>`) {
+		if strings.ContainsAny(object, `\:*?"|<>`) {
```

Comment:
missing some more changes in this file, please refer to https://github.com/minio/minio/pull/16856/files#diff-595284c3632e3f6c49ae1292914b68f6923c10e4ffa8175d039ce51eacc4646f